### DISCUSSION TITLE
Copilot suggests multiple lines in VSCode but prefers only single lines mostly in IntelliJ

### DISCUSSION THREAD

**Discussion by ionite34** [community/community#7046] [Copilot Conversations]
![image](https://user-images.githubusercontent.com/13956642/139553873-3542e375-ef06-40d6-a54c-53cd89ee9c8c.png)
vs.
![image](https://user-images.githubusercontent.com/13956642/139553896-eaa7617f-a30a-4676-a4cc-f5694a6b3e54.png)

**Comment by jansorg** [comment]
Thanks for reporting! Is this a Java file and inside a method?

**Reply by ionite34** [reply]
Yes. These are the same java file and inside a method. If I keep accepting the lines intelliJ eventually suggests everything that was suggested in vscode.

Sometimes copilot in intelliJ also suggests full multi lines.

**Comment by martin-marko** [comment]
I've noticed this also in PyCharm. It works if I keep pressing tab and enter, but I'd rather have the full suggestion as is, that'd make a lot more sense, it's pointless like this. Example:

![image](https://user-images.githubusercontent.com/82279281/140419674-8d0eac5f-a1bc-4a31-82ca-ecd7747e9a97.png)

Meanwhile in VSC:

![image](https://user-images.githubusercontent.com/82279281/140419846-1f834108-31df-472f-994d-8bba723da470.png)

**Comment by maxfahl** [comment]
I'm having the same issue. Any updates on this? Would love to be able to get multi line suggestions, and I really don't want to swap to VSCode for only this feature. I'm getting no multi line suggestions whatsoever, no matter the context or syntax. Thanks in advance.

**Comment by palyaros02** [comment]
I have only single line suggestions in VSCode, but in PyCharm sometimes it works properly.

**Comment by cmp-nct** [comment]
see my reports and suggestion here: https://github.com/community/community/discussions/36052
Copilot Plugin needs a shortcut for
a) Suggest ONE line
b) Suggest MULTIPLE lines

**Comment by Liquidmasl** [comment]
just got autopilot, this seams to be still an issue in pycharm. 
I just tried to switch to VS code but the overhead involved is barely worth it...

any solution/change?
